extensions/cl_intel_mem_alloc_buffer_location.asciidoc

```diff
@@ -137,7 +137,6 @@ Modifications to the OpenCL API Specification
 | Identifies the ID of global memory partition to which the memory should be allocated. The range of legal values is implementation defined. If the value is not valid, or the implementation is unable to allocate memory in the requested memory type, an `CL_INVALID_PROPERTY` error will be emitted.
 |====
 
-[[cl_mem_info_intel]]

```

Comment:
Mostly a note for myself - I initially thought it would be better to rename this anchor vs. removing it entirely, but it isn't used anywhere in the extension document so it's OK to remove it.

Reply:
Thinking about this a bit more: while it's OK to remove the anchor in this particular case, we're likely to have an issue in the future where multiple extension documents legitimately want to use the same anchor.  Should we suggest (in the extension template, perhaps) to prefix or suffix anchor names with the extension name to avoid collisions?  Or is this not something we need to worry about right now?

Reply:
Agree we will need a solution to this problem going forward. I've added a task on the newly created issue to track automatic generation (https://github.com/KhronosGroup/OpenCL-Docs/issues/848). Prefixing anchor names with the extension name sounds like a good direction. I can rename both conflicting anchors in this PR if you want but I don't really want to rename all extension anchors here :).

Reply:
Yeah, no need to change anything here, this PR is good to go.  I'll resolve this conversation so there is no confusion :).